main.Rmd

```diff
@@ -248,23 +248,23 @@ The presented EU-MOHP dataset\cite{nolscher_eu-mohp_2021-1} has version v013.1.1
+To further assess the quality of the applied methodology in this study, a cross-comparison with the original MOHP dataset for the contiguous US by Belitz et al. (2019) was performed. Therefore, we reproduced parts of the original MOHP dataset by applying our methodology to the NHDPlusV2 dataset\cite{nhdplusv2-dataset}, which is the underlying dataset of the original MOHP dataset, and compared these reproduced results to the original dataset\cite{belitz_multiorder_2019}. As the methodology is analogous for all hydrologic orders and the values of all three measures (DSD, LP and SD) have the same dependencies (DD and DS), it is sufficient to cross-compare LP for a single hydrologic order. For visual purposes, the 7^th^ hydrologic order was selected. Accordingly, the reproduced dataset will be referred to as "Reproduced LP7" and the original as "Original LP7". Figure \@ref(fig:comparisondifferencepatchwork) shows a side-by-side comparison between the Original LP7 (a) and Reproduced LP7 (b). From visual inspection, the major patterns appear very similar on both maps. Differences can be mainly observed in proximity to the administrative borders to Canada and Mexico. These difference among some other minor ones are due to deviations from the original methodology. Although the methodology of the original MOHP dataset\cite{belitz_multiorder_2019} is generally well described, it was not possible to fully comprehend and reproduce all steps due to the source code not being publicly available. For this reason, the coastline used for the cross-comparison is not fully identical to the coastline used for the original dataset. Likewise, neither river networks from the two neighboring countries Canada and Mexico nor surface water bodies in general are included in the reproduced dataset. The regions most affected by these differences are excluded from the quantitative cross-comparison.
 Figure \@ref(fig:comparisondifferencepatchwork)c shows the absolute difference between both maps from Figures \@ref(fig:comparisondifferencepatchwork)a and b defined as 
 
 \begin{equation}
 \text{Absolute difference} = \frac{\text{Original LP7} - \text{Reproduced LP7}}{10,000}. (\#eq:eqreldif)
 \end{equation}
-
+<!--figure? Figure? -->
```

Comment:
a few inconsistencies with how package names are formatted in general, as well as upper/lowercase for these things